### ISSUE TITLE
[BUG]xinput1_4.dll missing

### ISSUE THREAD

**Issue by MagicOneFr** [microsoft/RdpGamepad#5]
**Describe the bug**
The gamepad isn't working and I have a message box on the local computer when I start the rdp session about xinput1_4.dll that is missing.
My local computer runs Windows 7.

**To Reproduce**
1. Start the RDP session

**Expected behavior**
We should have a version using xinput 1.3 for Windows 7.

Thanks.

**Please complete the following information:**
 - OS: Windows 7 for the local computer, Windows 10 for the remote one.

**Comment by jpflouret** [comment]
Created release 1.0.1 to support Windows 7&8.

**Comment by MagicOneFr** [comment]
Thank you very much for the update but the pad doesn't work for me.
On the client side with the version 1.01 I don't have the error at launch anymore. 
On the remote side I have installed the  ViGEmBus driver and the 1.01 receiver. I can see the small controller in the icons bar.
But nothing happens if I try to use my local gamepad. How can I check if the connection is ok? 
I can see a "Controller" in the peripheral devices of the the remote machine but if I access the parameters, it's empty. I'm not able to use the test panel.


Note : On the remote side vigembus complains about a new version being available but I can't see it on the website. I have 1.16.115

**Comment by jpflouret** [comment]
I am unable to reproduce your problem. I have tested Windows 7 both as a local computer and as a remote computer and both work correctly with an Xbox One controller as a local gamepad (I don't have any other gamepads available for testing).

ViGEm Bus does have a newer version of the drivers than the one you listed above. Version [1.16.116](https://github.com/ViGEm/ViGEmBus/releases/tag/setup-v1.16.116) is currently the latest.

With the RdpGamepadCleint-1.0.1 installed on the local Windows 7 computer I am able to connect to a remote computer and the controller "just works" as expected. When you connect to the remote computer, the receiver will add a virtual Xbox 360 Controller device that should be visible in Device Manager on the remote computer. Looks like this on a Windows 10 remote computer:
![image](https://user-images.githubusercontent.com/9397098/78595749-68fd0680-77ff-11ea-9793-75151be24e94.png)
And like this on a Windows 7 remote computer:
![image](https://user-images.githubusercontent.com/9397098/80291201-7d4a6a00-8700-11ea-9c10-16c7f63f9231.png)

I currently don't have a way to display connection status on the local computer. The local client always uses the first Xbox gamepad connected (device number 0) and there is really nothing to configure on the client.

If you could provide additional details about your setup I might be able to help you diagnose the issue.

**Comment by MagicOneFr** [comment]
It's working now.
I was unable to see the controller in the peripheral manager on the remote computer (windows10) event if the controller icon was present in the task bar.
![image](https://user-images.githubusercontent.com/54553512/80344974-b0a80880-8868-11ea-8147-c93b71ec8ef1.png)
So I have uninstalled ViGEm and I have installed the latest version. The ViGEn github page is confusing because when I click on the https://github.com/ViGEm/ViGEmBus/releases/latest link on the first page, I see the 1.16.115 version and not the .116
After that, I have restarted the receiver and now I can see the pad controller in the peripherals of the remote computer and it's working, I can play with my local pad on the remote computer :)
Thank you very much for the support.
